tests/test_claude_wrapper_hooks.py

```
@@ -215,6 +215,109 @@ def run_wrapper(
+            env["CMUX_PORT_RANGE"] = "10"
+            env["CMUX_SHELL_INTEGRATION"] = "1"
+            env["CMUX_SHELL_INTEGRATION_DIR"] = str(tmp / "shell-integration")
+            env["CMUX_SOCKET_PATH"] = socket_path
+            env["CMUX_SURFACE_ID"] = "surface:test"
+            env["CMUX_TAB_ID"] = "tab:test"
+            env["CMUX_WORKSPACE_ID"] = "workspace:test"
+            env["TERMINFO"] = str(tmp / "terminfo")
+            env["FAKE_REAL_ENV_LOG"] = str(env_log)
+            env["FAKE_REAL_ARGS_LOG"] = str(args_log)
+
+            proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)